src/js/fs.js

```diff
@@ -20,6 +20,10 @@ var util = require('util');
 var fsBuiltin = native;
 
 fs.exists = function(path, callback) {
+  if (!(util.isString(path)) && !(util.isBuffer(path))) {
+    throw new TypeError("Path should be a string or a buffer");
```

Comment:
Should we follow single-quoted for string

Reply:
@yorkie Good point. In fact, there had been no rule for quote for string. However, we are preparing the rule like https://github.com/Samsung/iotjs/blob/master/.eslintrc.js#L56. Perhaps we can fix them all at once with eslint.

Reply:
Okay, I'm fine with your proposal :)